os_windows_test.go

```diff
@@ -0,0 +1,120 @@
+// +build windows
+
+package ctxvfs
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestWindowsMount(t *testing.T) {
+
+	tmpDir, err := ioutil.TempDir("", "vfs-localfs")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmpDir)
+	if err = os.MkdirAll(tmpDir, 0700); err != nil {
+		t.Fatal(err)
+	}
+	tmpFile := filepath.Join(tmpDir, "foo")
+	if err = ioutil.WriteFile(tmpFile, []byte("bar"), 0600); err != nil {
+		t.Fatal(err)
+	}
+
+	fStat, _ := os.Stat(tmpFile)
+
+	tmpFile = filepath.ToSlash(tmpFile)
+	tmpDir = filepath.ToSlash(tmpDir)
+
+	volname := filepath.VolumeName(tmpFile)
+
+	var (
+		empty       = ""           // - empty root, OS("")
+		volume      = volname      // - volume name, OS("C:")
+		slashvolume = "/" + volume // - volume name prefixed with slash, OS("/C:")
+		dir         = tmpDir       // - directory, OS("C:/foo/bar")
+		dirslash    = tmpDir + "/" // - directory followed by slash, OS("C:/foo/bar/")
+		slashdir    = "/" + tmpDir // - directory prefixed with slash, OS("/C:/foo/bar")
+	)
+
+	var (
+		file           = tmpFile                             // absolute file path, C:/foo/bar
+		slashfile      = "/" + tmpFile                       // absolute file path prefixed with slash, /C:/foo/bar
+		relfile        = strings.TrimPrefix(tmpFile, volume) // path w/o volume name, /foo/bar
+		shortfile      = "foo"                               // relative path, foo
+		slashshortfile = "/foo"                              // relative path prefixed with slash, /foo
+	)
+
+	// keys are mount points,
+	// values indicate if OS.Stat(path) should succeed for a given path
+	tests := map[string]map[string]bool{
+		empty: {
```

Comment:
can you just write the strings here instead? I think the test case is easier to understand that way.